.github/workflows/update-protobufs.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update Protobufs
+on:
+  push:
+    branches:
+    - yaakov/auto-protos # temporary trigger during development
+  #schedule:
+  #- cron: '0 17 * * SUN'
+
+jobs:
+  update-protobufs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+
+    - name: Fetch latest protobufs
+      run: |
```

Comment:
There's a script you can call: https://github.com/SteamRE/SteamKit/blob/master/Resources/ProtobufGen/update.ps1

I don't think you need the cd/git pull?

Reply:
ooooh I didn't know about `--remote`